src/MetalPerformanceShaders/MPSCnnNeuron.cs

```diff
@@ -0,0 +1,15 @@
+﻿#if XAMCORE_2_0 || !MONOMAC
+using System;
+using XamCore.Metal;
+using XamCore.Foundation;
+
+namespace XamCore.MetalPerformanceShaders {
+	public partial class MPSCnnNeuronPReLU {
+		public unsafe MPSCnnNeuronPReLU (IMTLDevice device, float [] a, nuint count) : this (NSObjectFlag.Empty)
+		{
+			fixed (void* aHandle = a)
```

Comment:
I assume `count` is supposed to be the number of elements in the `a` array? If so, you can remove it from the signature, since you can get that value from the array itself.